test/fixtures/group-basic.json

```diff
@@ -18,10 +18,10 @@
   "output": {
     "user_id": "user-id",
     "time": 1388534400000,
-    "user_properties": {
+    "groups": {
```

Comment:
The output order has to change because of the way the `encode(JSON.stringify`()) orders the inputs.